iocore/net/UnixNet.cc

```diff
@@ -70,20 +71,30 @@ class InactivityCop : public Continuation
         continue;
       }
 
+      if (ne->default_inactivity_timeout_in == -1) {
+        // If no context-specific default inactivity timeout has been set by an
+        // override plugin, then use the global default.
+        Debug("inactivity_cop",
+              "vc: %p setting a the global default inactivity timeout of %d, next_inactivity_timeout_at: %" PRId64, ne,
```

Comment:
this is nit-picky but I think you can remove the "a"

Reply:
Good catch.